### PR TITLE
Added TAC Meeting Notes For 2024-07-11

### PR DESCRIPTION
**Pull request by riaankleinhans**
@bensternthal

### CODE REVIEW

File: TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+* Cosmin Aprodu introduced themselves.
+* Jason Rogers introduced themselves.
+* Three folks from Google / [Project Oak](https://github.com/project-oak/oak) introduced themselves.
+* Yulia Gontar from Superprotocol introduced themselves.
+* Anna Trikalinou introduced themselves.
+
+## Old Business
+
+DM reviewed the GRC and Data Cleanroom Project proposal from the last meeting. Mark Novak (MN) reviewed the NIST announcements from the previous meeting. DM mentioned that the use document was published yesterday.
```
Comment (bensternthal) [inline TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md:88]: should read "use case"

File: TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+Alex Fernandez (AF) asked if the token met the CCC definition of "hardware-based, attested Trusted Execution Environment"? AF asked DM if we should ask the folks at Google if this meets the definition.
+
+Discussion ensued.
+
+DM asked if someone from the TAC wanted to make a motion to vote on the project or if folks wanted more time to consider the project. AF stated that he would prefer more time and to discuss privately with TAC members before voting. DM stated that it would be kept on the agenda and encouraged TAC members to use existing channels to discuss. Fritz Alder and Manu Fontaine agreed that they would like more time for discussion. Vini Jaisal (VJ) offered to reach out to any members that require any further clarification or information.
+
+## TAC Objectives / TAC Goals
+
+DM introduced Henry Wang (HW) and Kenvin Hui (KH) to discuss TAC Goals
```
Comment (bensternthal) [inline TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md:106]: missing . after TAC Goals

File: TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+HW announced that they have an internal Attestation encrypted channel establishment library project base on C++ helper functions that they would be interested in open sourcing. They are still working on removing internal dependencies before it is ready. Aiming to open source it before the end of 2024.
```
Comment (bensternthal) [inline TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md:107]: base should be based

File: TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+DM welcomed the proposal and said the submission is welcome and does not have to be a finished product, leaving room for contributions.
+
+KH discussed starting a working group around open source code transparency for data center owner companies. They have already discussed this with the folks from Project Oak. AF asked if they have looked at the documentation around Asure code transparency service and how this would be different from this proposal.KH he briefly looked at it. The aim of the group would be to bring everyone’s code transparency work together. Started the working group and would aim to bring more roleplayers together.
```
Comment (bensternthal) [inline TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md:110]: Asure is Azure

Comment (bensternthal) [reply TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md:110]: Suggested rewording of the second part of this paragraph:

KH briefly reviewed it. KH explained that the aim of the group would be to consolidate everyone's code transparency efforts. KH then stated that the working group has been initiated, and he aims to involve more key stakeholders.

File: TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md (hunk 1 above)
Comment (riaankleinhans) [reply TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md]: fixed

File: TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md (hunk 3 above)
Comment (riaankleinhans) [reply TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md:107]: fixed

File: TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md (hunk 2 above)
Comment (riaankleinhans) [reply TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md]: fixed

File: TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md (hunk 4 above)
Comment (riaankleinhans) [reply TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md:110]: fixed.

File: TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+AF also indicated he would be interested in this work.
+
+DM pointed out that this idea is helpful, but it produces more goals to pursue rather than addressing goals that were already set by the TAC of 2024. He further proposed that KH can start with a blog post figuring out what is already available in the field. That could also be presented as a TAC Tech talk to get more community input about other areas where this is valid.
```
Comment (bensternthal) [inline TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md:113]: by the TAC for 2024.

That could also be presented as a TAC Tech talk to get more community input about other areas where this is valid.

suggested rewording of "That could also be presented as a TAC Tech talk to get more community input about other areas where this is valid."

in the field suggesting a TAC tech talk to solicit community feedback.

**Comment by bensternthal** [comment]
@riaankleinhans minor nits from me.

File: TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md (hunk 5 above)
Comment (riaankleinhans) [reply TAC/Meetings/2024/2024-07-11/CCC-TAC-Minutes-2024-07-11.md:113]: agree

**Pull request merged**